bundle/config/mutator/process_target_mode_test.go

```diff
@@ -179,6 +187,12 @@ func TestProcessTargetModeProduction(t *testing.T) {
 	bundle.Config.Resources.Experiments["experiment2"].Permissions = permissions
 	bundle.Config.Resources.Models["model1"].Permissions = permissions
 	bundle.Config.Resources.ModelServingEndpoints["servingendpoint1"].Permissions = permissions
+	bundle.Config.Resources.RegisteredModels["registeredmodel1"].Grants = []resources.Grant{
```

Comment:
We can remove this from the test here. We don't validate that the grants for production mode right now.